core/unit_test/openmp/TestOpenMP_SubView_c_all.cpp

```diff
@@ -10,3 +10,4 @@
 #include <openmp/TestOpenMP_SubView_c10.cpp>
 #include <openmp/TestOpenMP_SubView_c11.cpp>
 #include <openmp/TestOpenMP_SubView_c12.cpp>
+#include <openmp/TestOpenMP_SubView_c13.cpp>
```

Comment:
@hcedwar it doesn't look like this was repeated for the other backends (Serial and Cuda), so the PR must have only been tested for the OpenMP backend.